R/RLearner_regr_randomForest.R

```diff
@@ -66,6 +68,18 @@ trainLearner.regr.randomForest = function(.learner, .task, .subset, .weights = N
   } else {
     data = getTaskData(.task, .subset, target.extra = TRUE)
     m = randomForest::randomForest(x = data[["data"]], y = data[["target"]], ...)
+    ## ugly hack
+    ## if the formula interface is used the training data is saved, but not with the default interface
+    ## the training data is used when predict.type = "se" and calibration is not FALSE
+    if (.learner$predict.type == "se") {
+      if (is.null(.learner$par.vals$calibrate)) {
+        m$data = data[["data"]]
+      } else {
+        if (.learner$par.vals$calibrate) {
+          m$data = data[["data"]]
```

Comment:
Maybe @zmjones can explain better what happens but as far as I can see it it's because we need that data to be stored in the model `m` later in at least one `*StandardError` function.